core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -498,4 +498,17 @@ class EMConfig {
 
     @Cfg("Whether to print how much search done so far")
     var showProgress = true
+
+    @Experimental
+    @Cfg("Whether or not enable a search process monitor for archiving evaluated individuals and Archive regarding an evaluation of search"+
+            "This is only needed when running experiments with different parameter settings")
```

Comment:
is ". " missing after "search"? Ie, it looks like it would be printed as "searchThis"

Reply:
fixed :)